test/Test/Pos/Types/CoinSpec.hs

```diff
@@ -135,9 +132,6 @@ overflowIntegerCausesError =
 coinToIntegralToCoin :: C.Coin -> Property
 coinToIntegralToCoin = C.mkCoin . C.unsafeGetCoin .=. identity
 
-wordToCoinToWord :: Word64 -> Property
```

Comment:
I think @rockbmb will be unhappy with this change. You could rewrite this property using `==>`. Also it would be good to add a test that `mkCoin tooBigNumber` fails.